test/functional/JIT_Test/src/jit/test/jitserver/JITServerTest.java

```diff
@@ -490,4 +543,67 @@ public void testServerComesUpAfterClientAndGoesDownAgain() throws IOException, I
 		logger.info("Stopping client...");
 		destroyAndCheckProcess(client, clientBuilder);
 	}
+
+	public void testServerSCC() throws IOException, InterruptedException {
+                logger.info("running testServerSCC: INFO and above level logging enabled");
+
+                redirectProcessOutputs(clientBuilder, "testServerSCC.client");
+                redirectProcessOutputs(serverBuilder, "testServerSCC.server");
+
+                updateJITServerPort();
+
+                final Process server = startProcess(serverBuilder, "server");
+
+                Thread.sleep(SERVER_START_WAIT_TIME_MS);
+
+                Process client = startProcess(clientBuilder, "client");
+
+                logger.info("Waiting for " + CLIENT_TEST_TIME_MS + " millis.");
+                Thread.sleep(CLIENT_TEST_TIME_MS);
+
+                // Check if the SCC exists
+                if (checkCacheExists("test_jitscc")) {
+                        logger.info("Check if AOT methods exists in the cache test_jitscc::" + checkAOTCompileExists("test_jitscc"));
```

Comment:
Do we fail the test if no AOT methods exist in SCC?

Reply:
I have not added a fail if AOT methods do not exist. But can be added if we need this.